pkg/sfu/buffer/buffer.go

```diff
@@ -437,6 +439,7 @@ func (b *Buffer) Close() error {
 			}
 		}
 
+		b.readCond.Broadcast()
```

Comment:
Is there a reason for `Broadcast` instead of `Signal`? Both are fine. Just trying to understand if `Broadcast` is needed.

Reply:
They are the same here since we only have 1 goroutine to call the Read now, just make it safe for multiple readers.